example/gen/haberdasher_twirp.rb

```diff
@@ -1,7 +1,7 @@
 # Code generated by protoc-gen-twirp_ruby, DO NOT EDIT.
 require 'twirp'
 
-class HaberdasherService < Twirp::Service
-  PATH_PREFIX = "/twirp/examples.Haberdasher"
-  rpc :HelloWorld, Examples::HelloWorldRequest, Examples::HelloWorldResponse
+class Example::HaberdasherService < Twirp::Service
```

Comment:
Instead of this, it would make more sense to produce nested modules:
```
module Example
    HaberdasherService < Twirp::Service
    end
end
```

This way the request/response messages can be referenced without the module prefix, and the generated code would look a lot closer to the protoc generated code for the messages. Is this maybe too hard?